lxd/daemon_images.go

```diff
@@ -31,20 +31,21 @@ import (
 
 // ImageDownloadArgs used with ImageDownload.
 type ImageDownloadArgs struct {
-	ProjectName       string
-	Server            string
-	Protocol          string
-	Certificate       string
-	Secret            string
-	Alias             string
-	Type              string
-	SetCached         bool
-	PreferCached      bool
-	AutoUpdate        bool
-	Public            bool
-	StoragePool       string
-	Budget            int64
-	SourceProjectName string
+	ProjectName              string
+	Server                   string
+	Protocol                 string
+	Certificate              string
+	Secret                   string
+	Alias                    string
+	Type                     string
+	SetCached                bool
+	PreferCached             bool
+	AutoUpdate               bool
+	Public                   bool
+	StoragePool              string
+	Budget                   int64
+	SourceProjectName        string
+	DownloadFromExplicitCopy bool
```

Comment:
As the struct is called `ImageDownload` the `Download` in `DownloadFromExplicitCopy` is redundant and stammers, e.g. `ImageDownload{}.DownloadFromExplicitCopy = true`

Lets just call it `UserRequested`.

Reply:
this aligns with https://github.com/canonical/lxd/blob/main/lxd/instance/instance_interface.go#L97

Reply:
fixed